lmcache/v1/cache_controller/worker.py

```diff
@@ -242,6 +244,13 @@ async def handle_request(self):
                     serialized_ret_msg = msgspec.msgpack.encode(
                         PinWorkerRetMsg(num_tokens=num_pinned_tokens)
                     )
+                elif isinstance(request, UnpinWorkerMsg):
+                    self.lmcache_engine.lookup_unpin(
+                        request_id=request.worker_event_id,
```

Comment:
is that correct? @YaoJiayi

Reply:
This `worker_event_id` was not randomly generated, it is generated at `pin` and then pass to `unpin`